packages/mds-ingest-service/@types/index.ts

```diff
@@ -128,6 +128,14 @@ export interface ReadTripEventsQueryParams {
   provider_id?: UUID
 }
 
+export interface ReadDeviceEventsQueryParams {
+  skip?: UUID
```

Comment:
Wait...are our UUIDs ordered?

Reply:
UUIDs are alpha-numeric, so you can order by 'em, yep!

Reply:
Should've specified, does UUID generation correlate with insertion order. Anyway thanks for the clarification.